src/status_im/config.cljs

```diff
@@ -136,6 +136,9 @@
 ;; The port comes from your running status backend server.
 ;; If you run it by PORT=60000 make run-status-backend , then host will likely be 127.0.0.1:60000
 (goog-define STATUS_BACKEND_SERVER_HOST "")
+;; enable media server over https or http
```

Comment:
Maybe it would be beneficial to use `(if platform/android? ..) here to make the code coherent with the comment, wdyt?